conda-forge.yml

```
@@ -7,5 +7,7 @@ github:
   branch_name: main
   tooling_branch_name: main
 provider:
-  win: azure
+  linux_aarch64: default
+  # Wait until native builds work on Travis to enable this
+  # linux_ppc64le: default  win: azure
```

Rather than comment this out I was told to use `skip` like
```suggestion
  linux_ppc64le: default
```

https://github.com/conda-forge/openmeeg-feedstock/pull/56/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR18

---

Thanks Eric!

Interestingly there seems to be a typo in the change you linked to:

```
skip: true # [py<39 or ppc64le]]
```
There's one closing bracket to many!